packages/amplify-nodejs-function-template-provider/resources/lambda/appsync-todo/api_key/index.js.ejs

```diff
@@ -0,0 +1,76 @@
+<%= props.topLevelComment %>
+
+import { default as fetch, Request } from 'node-fetch';
+
+const ENV_VAR = process.env;
+const GRAPHQL_API_KEY = ENV_VAR[Object.keys(ENV_VAR).find(key => key.endsWith('GRAPHQLAPIKEYOUTPUT'))];
+const GRAPHQL_ENDPOINT = ENV_VAR[Object.keys(ENV_VAR).find(key => key.endsWith('GRAPHQLAPIENDPOINTOUTPUT'))];
```

Comment:
discussed with @ykethan, this can be updated to: `ENV_VAR['API_<%= props.providerContext.projectName.toUpperCase() %>_GRAPHQLAPIENDPOINTOUTPUT']` or 
`process.env.API_<%= props.providerContext.projectName.toUpperCase() %>_GRAPHQLAPIENDPOINTOUTPUT;`

Reply:
fixed and committed the new props